modules/cluster-logging-release-notes-5.0.1.adoc

```diff
@@ -1,9 +1,18 @@
 [id="cluster-logging-release-notes-5-0-1"]
 = OpenShift Logging 5.0.1
 
-This release includes Red Hat Bug Advisory, link:https://errata.devel.redhat.com/docs/show/TBD[RHBA-TBD Errata Advisory for Openshift Logging 5.0.TBD].
+This release includes Red Hat Bug Advisory, link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 for OpenShift Logging Bug Fix Release (5.0.1)].
 
 [id="openshift-logging-5-0-1-bug-fixes"]
 == Bug fixes
 
-* If you did not set `.proxy` in the cluster installation configuration, and then configured a global proxy on the installed cluster, a bug prevented Fluentd from forwarding logs to Elasticsearch. To work around this issue, in the proxy/cluster configuration, set `no_proxy` to `.svc.cluster.local` so it skips internal traffic. The current release fixes the proxy configuration issue. Now, if you configure the global proxy after installing an OpenShift cluster, Fluentd forwards logs to Elasticsearch. (link:https://issues.redhat.com/browse/LOG-1187[*LOG-1187*])
+* Previously, logs were not sent to managed storage when legacy log forwarding was enabled. This happened because the internal generation of the log forwarding configuration improperly made an either/or decision for log forwarding versus legacy log forwarding. The current release fixes this issue: Logs are sent to managed storage when the `logStore` is defined in the `ClusterLogging` instance. Additionally, logs are sent to legacy log forwarding when enabled regardless of whether a managed logstore is enabled or not.
+(link:https://issues.redhat.com/browse/LOG-1172[*LOG-1172*], link:https://bugzilla.redhat.com/show_bug.cgi?id=1791251[BZ#1911477])
```

Comment:
It looks like the bugzilla link should be bolded (to match the other bug fix links)